prime-router/examples/data-lineage/lineage-report-stored-procedures.sql

```diff
@@ -0,0 +1,85 @@
+-- Stored procedures that do cool lineage queries on report_files
```

Comment:
These functions will now be permanently in the database, so they don't need to be manually loaded any more.

Reply:
Even better! Removed.